extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/YoutubeParsingHelper.java

```diff
@@ -96,7 +96,7 @@ private static boolean isGoogleURL(String url) {
         try {
             final URL u = new URL(url);
             final String host = u.getHost();
-            return host.startsWith("google.") || host.startsWith("m.google.");
+            return host.startsWith("google.") || host.startsWith("m.google.") || host.startsWith("www.google.");
```

Comment:
```suggestion
            return host.startsWith("google.") || host.startsWith("m.google.")
                    || host.startsWith("www.google.");
```

So you are in the 100 characters limit.